lib/api/pulsar/pulsar_tenant_api.dart

```diff
@@ -68,4 +68,17 @@ class PulsarTenantApi {
     }
     return response.data!;
   }
+
+  static Future<TenantInfoResp> getTenantInfoResp(
```

Comment:
can we use
```suggestion
  static Future<TenantInfoResp> getTenant(
``` ?
can we remove `getTenantInfo` method?